### DISCUSSION TITLE
The configuration of customResponseFields in smart-doc files does not take effect.

### DISCUSSION THREAD

**Discussion by chenyu3129** [TongchengOpenSource/smart-doc#768] [Q&A] [answered]
Premise: The structure of the response returned by the interface is either Response class or SingleResponse class, and Response is the parent class of SingleResponse. Both classes exist in a third-party library and the source files cannot be obtained.

### The configuration of the smart-doc.json file is as follows：
```
"customResponseFields": [
    {
      "name": "errCode",
      "ownerClassName": "com.alibaba.cola.dto.Response",
      "desc": "错误响应代码",
      "value": ""
    }
]
```

### The interface returned by the SingleResponse class generates the following response parameter results:
![企业微信截图_20240320145446](https://github.com/TongchengOpenSource/smart-doc/assets/31236415/57ddb2e9-6ed1-4aff-a0a1-4354da888dbb)

### The interface returned by the Response class generates the following response parameter results:
![企业微信截图_20240320145900](https://github.com/TongchengOpenSource/smart-doc/assets/31236415/1523ff84-448c-4ed5-93dc-8b030cc6d274)

**Comment by shalousun** [comment, marked as answer]
@chenyu3129 could reference https://smart-doc-group.github.io/#/zh-cn/faq/faq?id=%e6%b3%a8%e9%87%8a%e6%80%8e%e4%b9%88%e6%8f%90%e5%8f%96%e4%b8%8d%e5%88%b0%e5%95%8a%ef%bc%9f

The English document is available for reference. https://smart-doc-group.github.io/#/faq/faq?id=why-can39t-i-extract-the-comments
